spec/uvo_scraper_spec.rb

```diff
@@ -0,0 +1,44 @@
+require 'spec_helper'
+require "./uvo_scraper"
+
+describe UvoScraper do
+  let(:scraper) { UvoScraper.new }
+
+  describe ".issue_ready?" do
+    it "returns true if new issue is present" do
+      stub_request(:any, UvoScraper::NEW_ISSUE_URL).
+        to_return(:body => File.new('./spec/support/fixtures/new_issue_uvo_page.html'), :status => 200)
+
+      Timecop.freeze(Date.new(2016, 1, 25)) do
+        expect(scraper.issue_ready?).to eq true
+      end
+    end
+
+    it "returns false when new issue is missing" do
+      stub_request(:any, UvoScraper::NEW_ISSUE_URL).
+        to_return(:body => File.new('./spec/support/fixtures/new_issue_missing_uvo_page.html'), :status => 200)
```

Comment:
Line is too long. [111/80]<br>Use the new Ruby 1.9 hash syntax.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.